src/Stryker.Core/Stryker.Core/Instrumentation/IfInstrumentationEngine.cs

```diff
@@ -35,7 +37,7 @@ protected override SyntaxNode Revert(IfStatementSyntax ifNode)
     {
         if (ifNode.Else?.Statement is BlockSyntax block)
         {
-            return block.Statements.Count == 1 ? block.Statements[0] : block;
+            return (block.Statements.Count == 1 ? block.Statements[0] : block).WithTriviaFrom(ifNode);
```

Comment:
Does this always work? In some mutators, we mutate sub-nodes, so we might remove trivia there, but not re-add them here. Is that an issue?

Reply:
see previous comment: there shall be no sub node mutation. Otherwise it will not work; I failed to find a way to support both sub node mutations AND general trivia support.